### ISSUE TITLE
Use JSON for the TOC file, not CSV

### ISSUE THREAD

**Issue by ricksbrown** [peakcontent/nothingCMS#2]
Suggestion: it would be more "orthodox" to use JSON rather than CSV.

**Comment by peakcontent** [comment]
I agree, except one of the motivators for the 'CMS' is that it needs to be _really_ simple (specifically, for whoever maintains the documentation into the future...). 

I wanted something that didn't require learning a CMS structure (for example, DokuWiki) or Markdown and also something that, if the future documentor is totally HTML naive, would allow them to place content onto a web page without touching CSS and HTML.

What do you think - if I go down the JSON path then it's already veering towards becoming it's own system with new knowledge requirements (I've had a good crack at creating and editing JSON schema and it isn't something that happens straight out of the box...)?

That said, I suppose it could identify if the file is CSV or an object and parse it accordingly...

**Comment by ricksbrown** [comment]
I wouldn't recommend either/or, I'd make a  decision either way. Based on your reasoning I'd suggest for now running with CSV and come back to this point down the track.

I'm not entirely convinced that CSV is any easier than JSON but maybe I'm too comfortable with it to be a good judge. Of course XML is the most human readable of all but XML somehow lost the battle for the web,

Perhaps a simple UI to generate the toc file would be in order? This means the user would not need to know either CSV or JSON. Could be a simple web frontend maybe?

Alternatively, at some point (really soon) you are going to need a build phase and that could include a CSV (or XML) to JSON step.
